### PR TITLE
comments on existing codes of high assembly cell project

### PR DESCRIPTION
**Pull request by vincentliuk**
@subutai

### CODE REVIEW

File: launcher [hunk 1]
```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
```
Comment (subutai) [inline launcher]: What is this file for? Is it needed?

File: projects/high_order_cell_assembly/analyze_spencer_data.m [hunk 2]
```diff
@@ -36,10 +36,11 @@
 % %         load dataToSubutai_102616/Combo3_V1.mat
 % end
 
-load('/Users/ycui/Documents/SpencerData/data/Combo3_V1andAL.mat')
-spiketrain = Combo3_V1andAL(3, 1).spiketrain;
+load('./data/Combo3_V1andAL.mat')
```
Comment (subutai) [inline projects/high_order_cell_assembly/analyze_spencer_data.m]: For each of these files, please add comment in the top stating the purpose of the file.

Comment (subutai) [inline projects/high_order_cell_assembly/analyze_spencer_data.m]: Add comment showing format of this data file.

File: projects/high_order_cell_assembly/src/calculate_occurance_time.m [hunk 3]
```diff
@@ -23,7 +23,7 @@
 function [binEdges, timeJitterDistribution, timeInMovieDist, occurance_time] = calculate_occurance_time(frequency, numFramesPerStim, spikeMat, binaryWords, numCoactive)
 
 %% check occurence of high-order patterns
-idx = find(frequency >=2);
+idx = find(frequency >=2); % the frequency of spikings
```
Comment (subutai) [inline projects/high_order_cell_assembly/src/calculate_occurance_time.m]: Filename is misspelled. Should be "occurrence"

**Comment by subutai** [comment]
@vincentliuk Left a few small comments

**Comment by vincentliuk** [comment]
@subutai I have modified by your comments, and committed again.

**Pull request merged**